pom.xml

```diff
@@ -887,6 +890,27 @@
         </plugins>
     </build>
     
+    <profiles>
```

Comment:
Hi @zhfeng 
Could you fix the indent of the tags?

Reply:
Done